frodo.js

```diff
@@ -5,24 +5,23 @@
  * for the extras/ plugins if they are loaded into a presentation.
```

Comment:
SCRIPT-Consider extracting this repeated logic into a separate function to avoid duplication.